dask_cuda/explicit_comms/dataframe/shuffle.py

```diff
@@ -187,15 +197,8 @@ def partition_dataframe(
     partitions
         Dict of dataframe-partitions, mapping partition-ID to dataframe
     """
-    if column_names[0] != "_partitions" and hasattr(df, "partition_by_hash"):
-        return dict(
-            zip(
-                range(npartitions),
-                df.partition_by_hash(
-                    column_names, npartitions, keep_index=not ignore_index
-                ),
-            )
-        )
+    # TODO: Use `partition_by_hash` if/when dtype-casting is added
+    # (See: https://github.com/rapidsai/dask-cuda/pull/1356)
```

Comment:
The link refers to this PR, what is there exactly to see here for additional information on this TODO?

Reply:
Ah good catch! I was planning to file an issue to add a `cast_dtype` argument to `partition_by_hash` (similar to the argument added to `partitioning_index` in https://github.com/dask/dask/pull/10705). I'll update the link before this gets merged.